modules/metallb-installing-using-web-console.adoc

```diff
@@ -16,7 +16,9 @@ As a cluster administrator, you can install the MetalLB Operator by using the {p
 
 . In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
 
-. Search for the MetalLB Operator, then click *Install*.
+. Scroll or type a keyword into the *Filter by keyword* box to find the Operator you want. For example, type `metallb` to find the MetalLB Operator.
```

Comment:
I think it might make more sense to ask the user to scroll as an option after mentioning they can type a keyword. I think it would eliminate some confusion, but up to you:
```suggestion
. Type a keyword into the *Filter by keyword* box or scroll to find the Operator you want. For example, type `metallb` to find the MetalLB Operator.
```